resources/lang/pt-br/errors.php

```diff
@@ -30,11 +30,14 @@
     ],
     'community' => [
         'slack' => [
-            'not-eligible' => 'Sua conta não segue os requisitos para um convite do Slack.',
-            'slack-error' => 'Um erro ocorreu nos servidores do Slack. Tente novamente em alguns minutos.',
+            'not-eligible' => 'A sua conta não é elegível para um convite do Slack.',
```

Comment:
The verb "segue" seems out of place in comparison to, for example, "atinge" or "cumpre" in this sentence.

But I personally think it sounds better the way I translated it (it's also closer to the original).